handbook/people-ops/spending-company-money.md

```diff
@@ -1,3 +1,67 @@
 # Spending company money
+Please see below for a list of reimbursable expenses (all in USD).
+
+## One-time expenses
+**Laptop (engineers):** $3500
+
+**Laptop (non-engineers):** $2000
+
+**Laptop coverage:** $380
+
+**Monitor:** $380
+
+**Headphones:** $200
+
+**Keyboard:** $130
+
+**Mouse or trackpad:** $80 or $130
+
+**USB-C Adapter:** $70
+
+**HDMI/monitor cable:** $15
+
+**Laptop stand:** $40
```

Comment:
What would it take to give more flexibility to team mates rather than specifying amounts for each possible hardware component they might need in their work environment? Imagine that team mate 1 doesn't want or need and laptop stand nor a desk, but would rather have a larger monitor. Team mate 2 wants the opposite. Is there a reason we can't accommodate these preferences?